cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -59,10 +59,34 @@ class Robot(BaseSalesforceTask):
         "debug": {
             "description": "If true, enable the `breakpoint` keyword to enable the robot debugger"
         },
+        "ordering": {
+            "description": (
+                "Path to a file which defines the order in which parallel tests are run. "
+                "This maps directly to the pabot option of the same name. It is ignored "
+                "unless the processes argument is set to 2 or greater."
+            ),
+        },
         "processes": {
-            "description": "*experimental* Number of processes to use for running tests in parallel. If this value is set to a number larger than 1 the tests will run using the open source tool pabot rather than robotframework. For example, -o parallel 2 will run half of the tests in one process and half in another. If not provided, all tests will run in a single process using the standard robot test runner.",
+            "description": (
+                "*experimental* Number of processes to use for running tests in parallel. "
+                "If this value is set to a number larger than 1 the tests will run using the "
+                "open source tool pabot rather than robotframework. For example, -o parallel "
+                "2 will run half of the tests in one process and half in another. If not "
+                "provided, all tests will run in a single process using the standard robot "
+                "test runner. See https://pabot.org/ for more information on pabot."
```

Comment:
Note to self: we should textwrap.dedent the option descriptions so that it's possible to put them in a triple-quoted block rather than breaking lines like this.